src/irclog2html/irclog2html.py

```diff
@@ -571,6 +572,7 @@ def head(self, title, prev=('', ''), index=('', ''), next=('', ''),
 <html>
 <head>
   <meta http-equiv="Content-Type" content="text/html; charset=%(charset)s" />
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Technically, XHTML style should be using ` />` for standalone tags.

(I'm not sure I care, XHTML is dead.)

Reply:
(I fixed it after merging in the end.)